README.md

```diff
@@ -44,7 +27,8 @@ Once the test runner has finished, its results will be written to `<output-dir>/
 To run the tests to verify the behavior of the test runner, do the following:
 
 1. Open a terminal in the project's root
-2. Run `./bin/run-tests.sh`
+2. Run `./bin/setup-locally.sh`
+3. Run `./bin/run-tests.sh`
```

Comment:
Running step 2 I get: 

```
➜  gleam-test-runner git:(create) ./bin/setup-locally.sh
Downloading packages
 Downloaded 2 packages in 0.84s
```

But when trying step 3 the `jq` dependency is needed:

```
➜  gleam-test-runner git:(create) ./bin/run-tests.sh
all_fail: testing...
bin/run.sh: line 78: jq: command not found
```

Just wondering if this is the expected behaviour.

Reply:
When running tests using Docker I get this output: 

```
➜  gleam-test-runner git:(create) ./bin/run-tests-in-docker.sh
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
all_fail: testing...
--- /opt/test-runner/tests/all_fail/results.json
+++ /opt/test-runner/tests/all_fail/expected_results.json
@@ -1,5 +1,5 @@
 {
   "version": 1,
-  "status": "error",
-  "message": "error: HTTP error\n\nA HTTP request failed.\nThe error from the HTTP client was:\n\n    error sending request for url (https://repo.hex.pm/tarballs/gleam_stdlib-0.24.0.tar): error trying to connect: dns error: failed to lookup address information: Try again"
+  "status": "fail",
+  "message": "\u001b[0;31mF\u001b[0m\u001b[0;31mF\u001b[0m\nFailures:\n\n  1) all_fail_test:one_test/0\n\u001b[0;31m     Failure: ?assertEqual(<<111,111,112,115>>, Actual)\n       expected: <<\"oops\">>\n            got: <<\"one\">>\u001b[0m\n\u001b[0;36m     %% eunit_proc.erl:346:in `eunit_proc:with_timeout/3`\u001b[0m\n\u001b[0;36m     Output: \u001b[0m\n\u001b[0;36m     Output: \u001b[0m\n\n  2) all_fail_test:two_test/0\n\u001b[0;31m     Failure: ?assertEqual(<<111,111,112,115>>, Actual)\n       expected: <<\"oops\">>\n            got: <<\"two\">>\u001b[0m\n\u001b[0;36m     %% eunit_proc.erl:346:in `eunit_proc:with_timeout/3`\u001b[0m\n\u001b[0;36m     Output: \u001b[0m\n\u001b[0;36m     Output: \u001b[0m\n\n\u001b[0;31m2 tests, 2 failures\n\u001b[0m"
 }

all_fail: fail.
all_fail: /opt/test-runner/tests/all_fail/results.json does not match /opt/test-runner/tests/all_fail/expected_results.json

empty_file: testing...
--- /opt/test-runner/tests/empty_file/results.json
+++ /opt/test-runner/tests/empty_file/expected_results.json
@@ -1,5 +1,5 @@
 {
   "version": 1,
   "status": "error",
-  "message": "error: HTTP error\n\nA HTTP request failed.\nThe error from the HTTP client was:\n\n    error sending request for url (https://repo.hex.pm/tarballs/gleam_stdlib-0.24.0.tar): error trying to connect: dns error: failed to lookup address information: Try again"
+  "message": "error: Unknown module field\n   ┌─ ./test/empty_file_test.gleam:10:13\n   │\n10 │   empty_file.hello_world()\n   │             ^^^^^^^^^^^^\n\nThe module `empty_file` does not have a `hello_world` field."
 }

empty_file: fail.
empty_file: /opt/test-runner/tests/empty_file/results.json does not match /opt/test-runner/tests/empty_file/expected_results.json

partial_fail: testing...
--- /opt/test-runner/tests/partial_fail/results.json
+++ /opt/test-runner/tests/partial_fail/expected_results.json
@@ -1,5 +1,5 @@
 {
   "version": 1,
-  "status": "error",
-  "message": "error: HTTP error\n\nA HTTP request failed.\nThe error from the HTTP client was:\n\n    error sending request for url (https://repo.hex.pm/tarballs/gleam_stdlib-0.24.0.tar): error trying to connect: dns error: failed to lookup address information: Try again"
+  "status": "fail",
+  "message": "\u001b[0;32m.\u001b[0m\u001b[0;32m.\u001b[0m\u001b[0;31mF\u001b[0m\nFailures:\n\n  1) partial_fail_test:three_test/0\n\u001b[0;31m     Failure: ?assertEqual(<<111,111,112,115>>, Actual)\n       expected: <<\"oops\">>\n            got: <<\"three\">>\u001b[0m\n\u001b[0;36m     %% eunit_proc.erl:346:in `eunit_proc:with_timeout/3`\u001b[0m\n\u001b[0;36m     Output: \u001b[0m\n\u001b[0;36m     Output: \u001b[0m\n\n\u001b[0;31m3 tests, 1 failures\n\u001b[0m"
 }

partial_fail: fail.
partial_fail: /opt/test-runner/tests/partial_fail/results.json does not match /opt/test-runner/tests/partial_fail/expected_results.json

success: testing...
--- /opt/test-runner/tests/success/results.json
+++ /opt/test-runner/tests/success/expected_results.json
@@ -1,5 +1,4 @@
 {
   "version": 1,
-  "status": "error",
-  "message": "error: HTTP error\n\nA HTTP request failed.\nThe error from the HTTP client was:\n\n    error sending request for url (https://repo.hex.pm/tarballs/gleam_stdlib-0.24.0.tar): error trying to connect: dns error: failed to lookup address information: Try again"
+  "status": "pass"
 }

success: fail.
success: /opt/test-runner/tests/success/results.json does not match /opt/test-runner/tests/success/expected_results.json

syntax_error: testing...
--- /opt/test-runner/tests/syntax_error/results.json
+++ /opt/test-runner/tests/syntax_error/expected_results.json
@@ -1,5 +1,5 @@
 {
   "version": 1,
   "status": "error",
-  "message": "error: HTTP error\n\nA HTTP request failed.\nThe error from the HTTP client was:\n\n    error sending request for url (https://repo.hex.pm/tarballs/gleam_stdlib-0.24.0.tar): error trying to connect: dns error: failed to lookup address information: Try again"
+  "message": "error: Syntax error\n  ┌─ ./src/syntax_error.gleam:5:1\n  │\n5 │ }\n  │ ^ I was not expecting this.\n\nExpected one of: \"]\""
 }

syntax_error: fail.
syntax_error: /opt/test-runner/tests/syntax_error/results.json does not match /opt/test-runner/tests/syntax_error/expected_results.json

Some tests failed!
```

A bit puzzled by the last message `Some tests failed!`.

Reply:
> The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested

This is due to you running an arm machine (maybe a MacBook) and the image being compiled for amd64

Reply:
I've added notes on what needs to be installed, thanks!

Reply:
I've added notes on what needs to be installed, thanks!